### ISSUE TITLE
GCop 155

### ISSUE THREAD

**Issue by Karvan** [Geeksltd/GCop#27]
False alarm.

example:
```
  public static IEnumerable<Account> GetAll(AccountType type, Account linkedSolicitor = null)
        {
            var result = Database.GetList<Account>(a => a.AccountType == type);
            if (linkedSolicitor != null)
                result = result.Where(a => a.LinkedSolicitors.Contains(linkedSolicitor));
            return result;
        }
```

Warning:  GCop155	Should be written simply as a.LinkedSolicitors.Intersect(result)
